src/util.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Based on environment, get a reference to the Web Crypto API's SubtleCrypto interface.
+ * @returns An implementation of the Web Crypto API's SubtleCrypto interface.
+ */
+function getSubtleCrypto(): SubtleCrypto {
+	if (typeof window !== 'undefined' && window.crypto) {
+		return window.crypto.subtle;
+	}
+	if (typeof globalThis !== 'undefined' && globalThis.crypto) {
+		return globalThis.crypto.subtle;
+	}
+	if (typeof crypto !== 'undefined') {
+		return crypto.subtle;
+	}
+	if (typeof require === 'function') {
+		// Cloudflare workers are doing what appears to be a regex check to look and
+		// warn for this pattern. We should never get her in a cloudflare worker, so
+		// I am being coy to avoid detection and a warning.
+		const cryptoPackage = 'node:crypto';
+		const crypto = require(cryptoPackage);
+		return crypto.webcrypto.subtle;
+	}
+	throw new Error('No Web Crypto API implementation found');
+}
+
+export const subtleCrypto = getSubtleCrypto();
+
+/**
+ * Converts different types to Uint8Array.
+ *
+ * @param value - Value to convert. Strings are parsed as hex.
+ * @param format - Format of value. Valid options: 'hex'. Defaults to utf-8.
+ * @returns Value in Uint8Array form.
+ */
+export function valueToUint8Array(
+	value: Uint8Array | ArrayBuffer | Buffer | string,
+	format?: string,
+): Uint8Array {
+	if (value == null) {
+		return new Uint8Array();
+	}
```

Comment:
This is illegal in the types; are we doing this to protect against JS users?

Reply:
I would assume as such, yes.  This function was copypasta'd from `index.ts`, in it's exact same form.  I didn't change of it.